client/landing/stepper/declarative-flow/hundred-year-plan.ts

```diff
@@ -127,9 +127,9 @@ const HundredYearPlanFlow: Flow = {
 				case 'diy-or-difm':
 					if ( 'diy' === providedDependencies?.diyOrDifmChoice ) {
 						return navigate( hasSite ? 'new-or-existing-site' : 'setup' );
+					} else if ( providedDependencies?.nextStep === 'thank-you' ) {
+						return navigate( 'thank-you' );
 					}
-					// TODO: add VIP flow
-					return navigate( 'schedule-appointment' );
```

Comment:
The schedule-appointment step will be removed entirely in the next PR. We don't need it anymore since the Calendly is in a popup form and not a separate step, which was an initial idea.

Reply:
Done.